blazor/diagram/user-handle.md

```diff
@@ -705,6 +787,84 @@ The following code explains how to customize the connector fixed user handle.
 ```
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/Blazor-Diagram-Examples/tree/master/UG-Samples/UserHandle)
 
+## How to provide a template to fixed userhandle
+
+You can define fixed user handle style using template in [FixedUserHandleTemplate] at tag level. You can define separate templates for each node and connector by differentiating them based on their ID property. The template will be rendered when the PathData properties of the fixeduserhandle is not defined. However, if both path data and template are defined, the path data will take precedence, and the template will not be rendered. The following code explains how to define template for fixed user handle.
```

Comment:
You can define a fixed user handle style using a template in the [FixedUserHandleTemplate] at the tag level. You can define separate templates for each node and connector by differentiating them based on their ID property. The template will be rendered when the PathData properties of the fixeduserhandle is not defined. However, if both path data and template are defined, the path data will take precedence, and the template will not be rendered. The following code explains how to define a template for a fixed user handle.